lib/generators/light_service/generator_utils.rb

```diff
@@ -0,0 +1,45 @@
+module LightService
+  module Generators
+    module GeneratorUtils
+      def make_nested_dir(dir)
+        FileUtils.mkdir_p(dir)
+      end
+
+      def supported_test_frameworks
+        %i[rspec]
+      end
```

Comment:
Can we check what test framework is set up with the Rails project when the generator is executed? If there's no RSpec, maybe we could skip the test generator task as is.

Reply:
In order to do this, we need more components added into the gem from the rails ecosystem.

I was detecting the framework before, and bailing if it wasn't `rspec` but the minute I ran the tests in isolation, they failed because `Rails.application` was not available.

This was the code I was using to find the test framework which I removed.

```ruby
      def supported_test_frameworks
        %i[rspec]
      end

      def test_framework_supported?
        supported_test_frameworks.include? test_framework
      end

      def test_framework
        # Don't know a better way to get to this value, unfortunately.
        Rails.application.config.generators.options[:rails][:test_framework]
      end

      def must_gen_tests?
        options.tests? && test_framework_supported?
      end
```

It's possible that `Rails.application.config.generators.options[:rails][:test_framework]` is not the right way to find the test framework. I'll reach out and see how it's done.